subsys/bluetooth/host/hci_core.c

```diff
@@ -3133,6 +3133,34 @@ static void read_buffer_size_complete(struct net_buf *buf)
 #endif /* !defined(CONFIG_BT_CLASSIC) */
 #endif /* CONFIG_BT_CONN */
 
+#if defined(CONFIG_BT_CONN) || defined(CONFIG_BT_ISO)
+static void check_num_of_acl_iso_pkts(uint8_t num_acl_pkts, uint8_t num_iso_pkts)
```

Comment:
```suggestion
static void check_num_of_hci_data_bufs(uint8_t num_acl_pkts, uint8_t num_iso_pkts)
```

Suggest to refer to the sizes as the count of HCI ACL and ISO data buffers

Reply:
Imo current name better matches with the specs `Total_Num_LE_ACL_Data_Packets` and `Total_Num_ISO_Data_Packets` return parameters from LE Read Buffer Size command.